src/pykeen/sampling/negative_sampler.py

```diff
@@ -25,14 +25,23 @@ def __init__(
         self,
         triples_factory: TriplesFactory,
         num_negs_per_pos: Optional[int] = None,
+        filtered: bool = None,
+        corruption_scheme: Tuple[str] = None,
     ) -> None:
         """Initialize the negative sampler with the given entities.
 
         :param triples_factory: The factory holding the triples to sample from
         :param num_negs_per_pos: Number of negative samples to make per positive triple. Defaults to 1.
+        :param filtered: Whether proposed corrupted triples that are in the training data should be filtered.
+            Defaults to False.
+        :param corruption_scheme: What sides (h, r, t) should be corrupted. Defaults to head and tail (h, t).
         """
         self.triples_factory = triples_factory
         self.num_negs_per_pos = num_negs_per_pos if num_negs_per_pos is not None else 1
+        self.filtered = filtered if filtered is not None else False
+        self.corruption_scheme = corruption_scheme if corruption_scheme is not None else ('h', 't')
```

Comment:
The `or` operator is useful for making this shorter:

```suggestion
self.corruption_scheme = corruption_scheme or ('h', 't')
```

Reply:
Thanks, I didn't know that one 👍 Got to love Python ❤️

Reply:
Some more examples of the short-circuit evaluation of these operators. It's really powerful
```python
a = None or 5
assert a == 5
b = 5 or None
assert b == 5
c = 5 or 6
assert c = 5

a = None and 5
assert a is None
a = False and 5
assert a is False
a = [] and 5
assert a == []

b = 5 and None
assert b is None
b = 5 and True
assert b is True
```